arch/xtensa/core/crt1.S

```diff
@@ -4,34 +4,26 @@
  */
 
 /*
- * For most hardware / boards, this code sets up the C calling context
- * (setting up stack, PS, and clearing BSS) and jumps to __clibrary_start
- * which sets up the C library, calls constructors and registers destructors,
- * and calls main().
- *
  * Control arrives here at _start from the reset vector or from crt0-app.S.
  */
 
+#include <xtensa/simboard.h>
+#include <xtensa/simcall.h>
```

Comment:
I'm not sure we want to include these files for a real board.
Shouldn't we include them conditionally?